### PR TITLE
[xdoctest][task 79] reformat example code with google style in  `python/paddle/nn/functional/loss.py`

### PR DESCRIPTION
**Pull request by jinyouzhi**
### PR types

Others

### PR changes

Others

### Description

修改如下文件的示例代码，使其通过 `xdoctest` 检查：

- `python/paddle/nn/functional/loss.py`

预览：

- en: http://preview-paddle-pr-56901.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/jit/loss_en.html
- zh: http://preview-paddle-pr-56901.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/jit/loss_cn.html

### Related links
 
- #55629
- #55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -496,29 +500,35 @@ def edit_distance(
...
+            >>> distance, sequence_num = F.loss.edit_distance(input=input, label=label, input_length=input_len, label_length=label_len, normalized=False)
+            >>> print(distance)
+            Tensor(shape=[1], dtype=int64, place=Place(cpu), stop_gradient=True,
+                    [4])
+            >>> print(sequence_num)
+            Tensor(shape=[4, 1], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[3.],
+                    [2.],
+                    [4.],
+                    [1.]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[3.],
                     [2.],
                     [4.],
                     [1.]])
```

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -496,29 +500,35 @@ def edit_distance(
...
+            >>> distance, sequence_num = F.loss.edit_distance(input=input, label=label, input_length=input_len, label_length=label_len, normalized=True)
+            >>> print(distance)
+            Tensor(shape=[1], dtype=int64, place=Place(cpu), stop_gradient=True,
+                    [4])
+            >>> print(sequence_num)
+            Tensor(shape=[4, 1], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[0.75000000],
+                    [0.50000000],
+                    [1.        ],
+                    [0.25000000]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[0.75000000],
                     [0.50000000],
                     [1.        ],
                     [0.25000000]])
```

File: python/paddle/nn/functional/loss.py [hunk 3]
```diff
@@ -933,29 +947,36 @@ def hsigmoid_loss(
...
+            >>> paddle.set_device('cpu')
+            >>> paddle.seed(2023)
+
+            >>> input = paddle.uniform([4, 3])
+            >>> print(input)
+            Tensor(shape=[4, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[ 0.73167229,  0.04029441, -0.48078126],
+                    [ 0.81050646, -0.15199822, -0.18717426],
+                    [ 0.94041789,  0.48874724,  0.03570259],
+                    [ 0.46585739,  0.95573163, -0.91368192]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[ 0.73167229,  0.04029441, -0.48078126],
                     [ 0.81050646, -0.15199822, -0.18717426],
                     [ 0.94041789,  0.48874724,  0.03570259],
                     [ 0.46585739,  0.95573163, -0.91368192]])
```

File: python/paddle/nn/functional/loss.py [hunk 4]
```diff
@@ -933,29 +947,36 @@ def hsigmoid_loss(
...
+            >>> label = paddle.to_tensor([0, 1, 4, 5])
+            >>> num_classes = 5
+            >>> weight = paddle.uniform([num_classes - 1, 3])
+            >>> print(weight)
+            Tensor(shape=[4, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[-0.14721161,  0.43916738, -0.58377075],
+                    [-0.60536981, -0.23151302, -0.70793629],
+                    [-0.54572451, -0.10784978, -0.56684279],
+                    [ 0.35370791, -0.07079649,  0.84765708]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[-0.14721161,  0.43916738, -0.58377075],
                     [-0.60536981, -0.23151302, -0.70793629],
                     [-0.54572451, -0.10784978, -0.56684279],
                     [ 0.35370791, -0.07079649,  0.84765708]])
```

File: python/paddle/nn/functional/loss.py [hunk 5]
```diff
@@ -933,29 +947,36 @@ def hsigmoid_loss(
...
+            >>> out = F.hsigmoid_loss(input, label, num_classes, weight)
+            >>> print(out)
+            Tensor(shape=[4, 1], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[2.23681736],
+                    [1.97140026],
+                    [1.66425037],
+                    [2.54727197]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[2.23681736],
                     [1.97140026],
                     [1.66425037],
                     [2.54727197]])
```

File: python/paddle/nn/functional/loss.py [hunk 6]
```diff
@@ -1271,26 +1298,26 @@ def l1_loss(input, label, reduction='mean', name=None):
...
-            l1_loss = paddle.nn.functional.l1_loss(input, label, reduction='none')
-            print(l1_loss)
-            # Tensor(shape=[2, 2], dtype=float32, place=Place(gpu:0), stop_gradient=True,
-            #        [[0.20000005, 0.19999999],
-            #         [0.20000000, 0.79999995]])
+            >>> l1_loss = paddle.nn.functional.l1_loss(input, label, reduction='none')
+            >>> print(l1_loss)
+            Tensor(shape=[2, 2], dtype=float32, place=Place(cpu), stop_gradient=True,
+                    [[0.20000005, 0.19999999],
+                    [0.20000000, 0.79999995]])
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
                    [[0.20000005, 0.19999999],
                     [0.20000000, 0.79999995]])
```

File: python/paddle/nn/functional/loss.py [hunk 7]
```diff
@@ -1791,56 +1836,54 @@ def ctc_loss(
...
+                     [3.02332580e-01, 1.46755889e-01, 9.23385918e-02]],
+                    [[1.86260208e-01, 3.45560730e-01, 3.96767467e-01],
+                     [5.38816750e-01, 4.19194520e-01, 6.85219526e-01]],
+                    [[2.04452246e-01, 8.78117442e-01, 2.73875929e-02],
+                     [6.70467496e-01, 4.17304814e-01, 5.58689833e-01]],
+                    [[1.40386939e-01, 1.98101491e-01, 8.00744593e-01],
+                     [9.68261600e-01, 3.13424170e-01, 6.92322612e-01]],
+                    [[8.76389146e-01, 8.94606650e-01, 8.50442126e-02],
+                     [3.90547849e-02, 1.69830427e-01, 8.78142476e-01]]
+                ]), dtype="float32")
```
Comment (megemini) [inline python/paddle/nn/functional/loss.py]: 需要加 `...`

File: python/paddle/nn/functional/loss.py [hunk 8]
```diff
@@ -3669,20 +3732,22 @@ def triplet_margin_loss(
...
+            >>> print(loss)
+            Tensor(shape=[3], dtype=float32, place=Place(cpu), stop_gradient=True,
+                   [0.        , 0.57496595, 0.        ])
 
-            loss = F.triplet_margin_loss(input, positive, negative, margin=1.0, reduction='mean')
-            print(loss)
-            # Tensor(0.19165580)
+            >>> loss = F.triplet_margin_loss(input, positive, negative, margin=1.0, reduction='mean')
+            >>> print(loss)
+            >>> # Tensor(0.19165580)
```
Comment (megemini) [inline python/paddle/nn/functional/loss.py]: 这行可以删掉 ～

Comment (jinyouzhi) [reply python/paddle/nn/functional/loss.py]: nice catch

File: python/paddle/nn/functional/loss.py (hunk 7 above)
Comment (jinyouzhi) [reply python/paddle/nn/functional/loss.py]: thanks

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾
